### PR TITLE
Update token model

### PR DESCRIPTION
**Pull request by ioanacrant**
removed permissions, added token_type

### CODE REVIEW

**Comment by pfhayes** [comment]
people might be depending on `permissions` so we should probably leave it in

**Comment by pfhayes** [comment]
there is a `DeprecatedField` annotation as well that might be helpful

**Comment by pfhayes** [comment]
lgtm!

File: .cache/v/cache/lastfailed [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (pfhayes) [inline .cache/v/cache/lastfailed]: what's this?

Comment (ioanacrant) [inline .cache/v/cache/lastfailed]: I have no idea....

**Comment by pfhayes** [comment]
Released in 2.7.2

**Pull request merged**